lib/asciidoctor-diagram/ditaa/extension.rb

```diff
@@ -1,29 +1,12 @@
 require 'set'
 
 require_relative '../extensions'
-require_relative '../util/java'
+require_relative 'converter'
 
 module Asciidoctor
   module Diagram
-    # @private
     module Ditaa
-      OPTIONS = {
-          'scale' => lambda { |o, v| o << '--scale' << v if v },
-          'tabs' => lambda { |o, v| o << '--tabs' << v if v },
-          'background' => lambda { |o, v| o << '--background' << v if v },
-          'antialias' => lambda { |o, v| o << '--no-antialias' if v == 'false' },
-          'separation' => lambda { |o, v| o  << '--no-separation' if v == 'false'},
-          'round-corners' => lambda { |o, v| o  << '--round-corners' if v == 'true'},
-          'shadows' => lambda { |o, v| o  << '--no-shadows' if v == 'false'},
-          'debug' => lambda { |o, v| o  << '--debug' if v == 'true'},
-          'fixed-slope' => lambda { |o, v| o  << '--fixed-slope' if v == 'true'},
-          'transparent' => lambda { |o, v| o  << '--transparent' if v == 'true'}
-      }
-
-      JARS = ['ditaa-1.3.13.jar', 'ditaamini-0.11.jar'].map do |jar|
-        File.expand_path File.join('../..', jar), File.dirname(__FILE__)
-      end
-      Java.classpath.concat JARS
+      OPTION_KEYS = %w[scale tabs background antialias separation round-corners shadows debug fixed-slope transparent]
```

Comment:
Shouldn't we reference the `OPTIONS` array from DitaaConverter instead of duplicating it here?

Reply:
Yes I think it's a good idea :+1: